tests/Unit/Subscriber/Behavior/SubscriberTestCase.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager\Tests\Unit\Subscriber\Behavior;
+
+class SubscriberTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
An unfinished improvement I think. The subscriber tests share a lot of requirements, the idea is to reduce the boilerplate code.

I think this is a natural evolution, so despite the fact that there is nothing in this class we should keep it.